x-pack/test/plugin_functional/plugins/resolver_test/kibana.json

```diff
@@ -3,7 +3,12 @@
   "version": "1.0.0",
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "resolver_test"],
-  "requiredPlugins": ["embeddable"],
+  "requiredPlugins": [
```

Comment:
the resolver test plugin needs access to `securitySolution` since thats where Resolver lives today.